Product/Production/Adapters/General/CONNECTAdminGUI/src/main/webapp/certificateManager.xhtml

```diff
@@ -328,6 +334,167 @@
                         </div>
                     </p:tab>
                     <!-- END: TAB PANE - MANAGE TRUSTSTORE -->
+
+                    <!-- BEGIN: TAB PANE - IMPORTCERTTAB -->
+                    <p:tab id="#{tabBean.IMPORTCERTSTORETAB}" title="Import Wizard" closable="false">
+                        <p:tabView id="importCertTabView" styleClass="tab-pane active cssImportWizard"
+                            activeIndex="#{certificateBean.importWizardTabIndex}">
+                            <p:tab id="tabIdCreateCert" title="Create Certificate" closable="false">
+
+                                <div class="form-block center-block">
+                                    <h:form id="formCreateCertificate" class="form form-horizontal">
+                                        <p:messages id="errorMsgsCreateCertificate" closable="true">
+                                            <p:autoUpdate />
+                                        </p:messages>
+                                        <div class="content">
+                                            <h:outputText>Creating a certificate will generate an entirely new key pair in the keystore file. The existing server certificate will be deleted. Please proceed to "Create CSR" if the existing server certificate is to be used.</h:outputText>
```

Comment:
This needs to be worded better - which existing certificate?

Reply:
this is cut and paste from the mockup